test/conftest.py

```diff
@@ -170,16 +170,74 @@ def git_urls():
             'name': 'repo',
             'owner': 'owner',
         },
+        'https://example.com/owner/repo': {
+            'pathname': '/owner/repo',
+            'protocols': ['https'],
+            'protocol': 'https',
+            'href': 'https://example.com/owner/repo',
+            'resource': 'example.com',
+            'user': None,
+            'port': None,
+            'name': 'repo',
+            'owner':'owner',
+        },
+        'example.com:repo.git':{
+            'pathname': 'repo.git',
+            'protocols': [],
+            'protocol': 'ssh',
+            'href': 'example.com:repo.git',
+            'resource': 'example.com',
+            'user': None,
+            'port': None,
+            'name': 'repo',
+            'owner': None,
+        },
+        'git+https://example.com/owner/repo.git':{
+            'pathname': '/owner/repo.git',
+            'protocols': ['git', 'https'],
+            'protocol': 'https',
+            'href': 'git+https://example.com/owner/repo.git',
+            'resource': 'example.com',
+            'user': None,
+            'port': None,
+            'name': 'repo',
+            'owner': 'owner',
+        },
+        'https://example.com/repo':{
+            'pathname': '/repo',
+            'protocols': ['https'],
+            'protocol': 'https',
+            'href': 'https://example.com/owner',
```

Comment:
what is happening here??  how can this be `.../owner`.  Something must be wrong with the test suite to allow this value here??

Reply:
This still hasnt been addressed.  Bypassing this question doesnt fix it.